<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lqWelcomeDialog</class>
 <widget class="QDialog" name="lqWelcomeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to LidarView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelAboutImage">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../Components/Resources/lvComponents.qrc">:/lvComponents/Icons/AboutDialog.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="chooseModeWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Please choose one of LidarView interface:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxMode">
           <item>
            <property name="text">
             <string>Lidar Viewer</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Components/Resources/lvComponents.qrc">
              <normaloff>:/lvComponents/Icons/lvLidarViewerSetup.svg</normaloff>:/lvComponents/Icons/lvLidarViewerSetup.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Point Cloud Tool</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Components/Resources/lvComponents.qrc">
              <normaloff>:/lvComponents/Icons/lvPointCloudToolSetup.svg</normaloff>:/lvComponents/Icons/lvPointCloudToolSetup.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Advanced Mode</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Components/Resources/lvComponents.qrc">
              <normaloff>:/lvComponents/Icons/lvAdvancedModeSetup.svg</normaloff>:/lvComponents/Icons/lvAdvancedModeSetup.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelLidarViewer">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>The &quot;Lidar Viewer&quot; mode simplifies the interface to display only the essential elements for displaying lidar data. It is the ideal mode to select if you solely want to connect to a lidar (or read a pcap) and visualize the output data, record it or save to another format.
To apply filters and processing pipelines (including PCL wrapping, SLAM and other features, including the ones from plugins), we suggest to switch to &quot;Advanced Mode&quot;.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPointCloudTool">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>The &quot;Point Cloud Tool&quot; mode simplifies the interface by eliminating tools related to LiDAR temporal data and connection to a live sensor. It is designed to apply filters on static point clouds, such as the ones acquired by aerial LiDAR, Terrestrial Laser Scanners, or SLAMed sequences of mobile Laser Scanners.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAdvancedMode">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>The &quot;Advanced Mode&quot; integrates both the &quot;Lidar Viewer&quot; and &quot;Point Cloud Tool&quot; modes, along with additional ParaView tools. It represents the most comprehensive LidarView interface.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelChangeUI">
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Note that you can always switch modes after closing this window by clicking on the corresponding icons in the  &quot;User Interface Control&quot; toolbar.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="DoNotShowAgainButton">
        <property name="text">
         <string>Don't show this window again</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Components/Resources/lvComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lqWelcomeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lqWelcomeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
